examples/sponsoredTXs/src/constants.ts

```diff
@@ -0,0 +1,45 @@
+export const CONTRACT_SUB_INDEX = 0n;
+
+const WALLET_CONNECT_PROJECT_ID = '76324905a70fe5c388bab46d3e0564dc';
```

Comment:
Where does this come from?

Reply:
https://concordium.slack.com/archives/C037WKK99NE/p1679042051884009?thread_ts=1679041309.052459&cid=C037WKK99NE

The  WALLET_CONNECT_PROJECT_ID, is the ID allocated to the account that Concordium has with walletconnect. Other dApp developers should use the same number. It is probably just for grouping traffic or something. It is the ID for our wallets, so WC knows which app to open a connection with.

Reply:
Should this not be a constant defined in our libraries then?

Can you make an issue to have this added?

Reply:
https://github.com/Concordium/concordium-dapp-libraries/issues/30